Makefile

```diff
@@ -87,4 +96,4 @@ coveralls:
 
 circle-ci:
 	@${cidocker} make install-deps build-local check-local coveralls
-	make integration-tests
+	make integration-tests clean
```

Comment:
Good point.
What I am trying to do is to execute the `clean` when `sudo make` is run outside of `circle-ci`, natively.
M